src/fragments/lib/auth/js/social.mdx

```diff
@@ -177,6 +177,11 @@ function App() {
 }
 ```
 
+<Callout warning={true}>
+  You can also use the <a href="https://ui.docs.amplify.aws/react/components/authenticator#social-providers">Authenticator UI component</a> to add social sign in flow to your application. Visit <a href="https://ui.docs.amplify.aws/react/components/authenticator#social-providersr">Social Provider</a> section to
```

Comment:
Typo in the second link; you added another "r" to the end of "providers"

Reply:
Thanks for the catch! Fixed in f17ed91acb2fafb08a497a135aa60cd7352fc2ad.